code/modules/clothing/suits/utility.dm

```diff
@@ -32,12 +32,19 @@
 	resistance_flags = FIRE_PROOF
 
 /obj/item/clothing/suit/fire/firefighter
+	icon = 'icons/obj/clothing/suits/utility.dmi'
+	icon_override = 'icons/mob/clothing/suits/utility.dmi'
 	icon_state = "firesuit"
-	item_state = "firefighter"
-
+	item_state = "firesuit"
 	sprite_sheets = list(
-		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
-		)
+		"Tajaran" = 'icons/mob/clothing/species/tajaran/suits/utility.dmi',
+		"Unathi" = 'icons/mob/clothing/species/unathi/suits/utility.dmi',
+		"Vulpkanin" = 'icons/mob/clothing/species/vulpkanin/suits/utility.dmi',
+		"Vox" = 'icons/mob/clothing/species/vox/suits/utility.dmi',
+		"Grey" = 'icons/mob/clothing/species/grey/suits/utility.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suits/utility.dmi',
+		"Kidan" = 'icons/mob/clothing/species/kidan/suits/utility.dmi'
+	)
 
 /obj/item/clothing/suit/fire/heavy
```

Comment:
```suggestion
/obj/item/clothing/suit/fire/firefighter/heavy
```

Not sure where this shows up in game (if at all) but best to make it a subtype so it shares the sprites just in case it is spawned in. Also remove the "item_state"/"icon_state" variable to ensure this.

Reply:
The heavy firesuit is not used anywhere, but it uses the sprite of its parent, its not a great sprite but it has one. It can just be deleted but that's not related to this PR I don't think?

Reply:
If it is used absolutely nowhere in the game, you should be able to clean it up in this PR.

Reply:
Well its gone now.